script/Ciel/Day 13/13_01_02_CIEL13_7E.txt

```diff
@@ -496,37 +585,41 @@ Left at a loss for words, the boy finally comes to understand the source of his
 [sha:ae4706309155db9a087ef7f831b92b928e030007]{
 -- Page 398, Offset 36436.
 -- 己の力を制限無しで自由に使う真祖は脅威なだけで、そう不可思議なものではない。
-If it had only been the threat of an uninhibited True Ancestor using the full breadth of their powers, then there wouldn't have been anything to be surprised about.
+A True Ancestor exercising the true extend of their powers would be a threat,
+ but they'd still be a known quantity.
 }
 [sha:cd396559247c6dfa724cf77ea5e67b227018c883]{
 -- Page 398, Offset 36437.
 -- 過去、吸血衝動に負け暴走した低級の真祖を鎮めた事も一度ぐらいなら経験がある。
-In the past, he had at least one experience of suppressing a low-grade True Ancestor who had gone berserk because of their bloodsucking impulse.
+He's already dealt with a low-level True Ancestor that lost to their vampiric urges and went berserk once before.
 }
 [sha:2c0151bad4e5bc8ae36e5614ccc57dae44136654]{
 -- Page 399, Offset 36438.
 -- だが、これは前例がない。
-However, this is completely unprecedented.
+But this... this is completely unprecedented.
 }
 [sha:c1636c3bbb66a7a1ca04952d2af84b40a45ee79b]{
 -- Page 399, Offset 36439.
 -- この真祖は有り得ない故障をしてしまった。
-The True Ancestor has suffered an umimaginable malfunction.
+This True Ancestor has suffered an unimaginable malfunction.
 }
 [sha:0c994f13857421c3a21709a165e84531f46b8929]{
 -- Page 399, Offset 36440.
 -- よりにもよって、一番狂ってはいけない吸血鬼が、一番陥ってはいけない状況に落ちてしまった。
-To make matters worse, the one vampire who must never be allowed to run amok has lost her mind and fallen into the worst situation imaginable.
+To make matters worse, this vampire――
+who cannot be allowed to lose herself under %{i}any%{/i} circumstances――
+has lost herself under the worst circumstances imaginable.
```

Comment:
I feel like emphasizing that she would be the worst vampire to fall in the state is imperative for this sentence to work.
That and like, the repetition is kinda important, but it's about the severity of the situation, so that's what should be emphasized:

Something like this.
>To make matters worse, 
the absolute worst vampire possible to ever lose control, 
has now fallen into the absolute worst state possible for her.

I do feel like it can be smoothened out somewhat though. Maybe change "to make matter worse" to avoid to many worse/worst. Uhh... "out of everything that could go wrong"? Kinda lengthy...